packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodForm.tsx

```diff
@@ -29,25 +29,20 @@ const PaymentMethodForm = ({ onAdd, onResetFormState, ...rest }: TPaymentMethodF
         reset,
     } = useForm({ mode: 'all' });
     const [isModalOpen, setIsModalOpen] = useState(false);
-    const { actionType, selectedPaymentMethod, title } = rest.formState || {};
-
+    const { actionType, selectedPaymentMethod, title } = rest.formState;
```

Comment:
`formState` is always defined since it's a required prop. I don't know what I was thinking 😄